sdk-python/examples/interruption/example_interruption.py

```diff
@@ -0,0 +1,59 @@
+import asyncio
+import logging
+from pathlib import Path
+
+import littlehorse
+from littlehorse.config import LHConfig
+from littlehorse.worker import LHTaskWorker
+from littlehorse.workflow import ThreadBuilder, Workflow
+
+logging.basicConfig(level=logging.INFO)
+
+
+def get_config() -> LHConfig:
+    config = LHConfig()
+    config_path = Path.home().joinpath(".config", "littlehorse.config")
+    if config_path.exists():
+        config.load(config_path)
+    return config
+
+
+def get_workflow() -> Workflow:
+    def my_interrupt_handler(thread: ThreadBuilder) -> None:
+        thread.execute("interrupt-handler")
+
+    def my_entrypoint(thread: ThreadBuilder) -> None:
+        thread.add_interrupt_handler("interruption-event", my_interrupt_handler)
+        thread.sleep(30)
```

Comment:
30 seconds is good. We should describe what this Workflow does in the README. Say "the parent thread sleeps 30 sec, we can interrupt it while it's sleeping, and it creates child thread, etc"